api/tests/unit/infrastructure/validators/grecaptcha-validator_test.js

```diff
@@ -0,0 +1,116 @@
+const {describe, it, expect, sinon} = require('../../../test-helper');
+const gRecaptcha = require('../../../../lib/infrastructure/validators/grecaptcha-validator');
+const request = require('request');
+const logger = require('../../../../lib/infrastructure/logger');
+
+const INVALID_RECAPTCHA_TOKEN = 'INVALID_RECAPTCHA';
+const RECAPTCHA_TOKEN = '03AOPBWq981qDlqqeuJvRRPOdV_jUsJKV7K9y8EdroTMzC50kWInTzGqEdKAhaSRozRS2Z84XzrpmcGmmF1qxC06lGPxbBfYOon_dVIie98Dr-sclrLJ69D6C3E5r9DLxNvKZjSF9cEmp7HdJzHR1Bw5-fqoqAw-9daY05zEU7abA6Wr59RPs8UvU1QnWXACcbIDW64F723EeAOWBuq14vUaYJRWGEhSbucM7uL7Uc9YbErVbJoVmH5YwVPuQiHBv9F39UBUC7qIaSKbruIy6lD7pKL0_Acyy_BGkaTSavHAdTBG_b091ex6-XheE8UigM2YeTKnch_6sandf0lmt47SCvwFlvBBMQMWJL2h8fpXfijP2kTEQiMlIhdOEslkqDykE6hCMtZLsE';
```

Comment:
pas la peine de mettre tout un vrai token ; un truc du genre : `const VALID_RECAPTCHA_TOKEN = 'a-valid-recaptch-token-should-be-a-string-of-512-numalpha-characters'` me paraîtrait plus parlant pour se faire une idée de l'algo.